rockcraft.yaml

```diff
@@ -12,10 +12,24 @@ platforms:
 services:
   kratos:
     override: replace
-    command: kratos serve
+    command: /bin/sh -c "kratos serve all 2>&1 | tee -a /aba.txt"
```

Comment:
Is there a reason for this specific file name or can we put `/var/log/kratos.log`?

Reply:
Leftover from tests. I removed it